shop/models/product.py

```diff
@@ -353,6 +357,8 @@ def update_search_index(self):
         of the product.
         """
         documents = elasticsearch_registry.get_documents([ProductModel])
+        if len(documents) == 0:
```

Comment:
`len(documents)` would already evaluate as `False`, no need to check for comparison to `0`